presto-docs/src/main/sphinx/functions/noisy.rst

```diff
@@ -123,6 +123,29 @@ is supported via the Sketch-Flip-Merge (SFM) data sketch [Hehir2023]_.
         Unlike :func:`approx_set`, this function returns ``NULL`` when ``col`` is empty.
         If this behavior is undesirable, use :func:`coalesce` with :func:`noisy_empty_approx_set_sfm`.
 
+.. function:: noisy_approx_set_sfm_from_index_and_zeros(col_index, col_zeros, epsilon, buckets[, precision]) -> SfmSketch
+
+    Returns an SFM sketch of the input values in ``col_index`` and ``col_zeros``. This is similar to
+    :func:`noisy_approx_set_sfm` except that function calculates a ``Murmur3Hash128.hash64()`` of ``col``,
+    and calculates the SFM PCSA bucket index and number of trailing zeros as described in [FlajoletMartin1985]_,
+    whereas in this function the caller must explicitly calculate the hash bucket index
+    and zeros themselves and pass them as arguments ``col_index`` and ``col_zeros``.
+
+    - ``col_index`` (bigint) must be in the range ``0..buckets-1``
+    - ``col_zeros`` (bigint) must be in the range ``0..64``. If it exceeds ``precision``, it
+      will be cropped
+    - ``epsilon`` (double) is a positive number that controls the level of noise in
+      the sketch, as described in [Hehir2023]_. Smaller values of epsilon correspond
+      to noisier sketches.
+    - ``buckets`` (int) is the number of buckets in the SFM PCSA sketch as described in [Hehir2023]_.
+    - ``precision`` (int) defaults to 24.
+
+    .. note::
```

Comment:
No nit, just want to say this is a great note! It emphasizes potentially undesirable behavior that a reader might overlook, and it also recommends an alternate choice for the reader to consider.